Index: app/src/main/java/com/example/dsmabsen/helper/Constans.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.dsmabsen.helper\r\n\r\nobject Constans {\r\n    const val BASE_URL = \"https://ngastaa.com/api/\"\r\n    const val IMAGE_URL = \"https://ngastaa.com/\"\r\n    const val TAG = \"fauzanapp\"\r\n    const val USER_TOKEN = \"user_token\"\r\n    const val PREFS_TOKEN_FILE = \"prefs_token_file\"\r\n    const val USER_SAVED = \"user\"\r\n}
===================================================================
diff --git a/app/src/main/java/com/example/dsmabsen/helper/Constans.kt b/app/src/main/java/com/example/dsmabsen/helper/Constans.kt
--- a/app/src/main/java/com/example/dsmabsen/helper/Constans.kt	
+++ b/app/src/main/java/com/example/dsmabsen/helper/Constans.kt	
@@ -1,6 +1,8 @@
 package com.example.dsmabsen.helper
 
 object Constans {
+//    const val BASE_URL = "https://ngastaa.com/api/"
+//    const val IMAGE_URL = "https://ngastaa.com/"
     const val BASE_URL = "https://ngastaa.com/api/"
     const val IMAGE_URL = "https://ngastaa.com/"
     const val TAG = "fauzanapp"
